api/v1/cube.js

```diff
@@ -96,7 +42,79 @@ const decode = (params) => {
     return { center: false };
   } catch (e) {}
 
-  throw Error(`Unsupported interface for cube: ${JSON.stringify(params)}`);
+  const { roundRadius = undefined, resolution = 8 } = params[0];
+  const roundParams = { round: false };
+  if (typeof roundRadius === 'number' && typeof resolution === 'number') {
+    Object.assign(roundParams, { round: true, roundingRadius: roundRadius, roundingFaces: resolution });
+  }
+
+  const decoded = (() => {
```

Comment:
I thought this was defining a function until I read all the way to the end. :)

Can we separate the function definition from its application?